components/ModalView/ModalViewGene.vue

```diff
@@ -148,7 +148,9 @@
             this.data = data;
           })
           .catch(function (error) {
```

Comment:
細かいですが、`error`に`_`をつけた方が丁寧です。「変数を自動的に受け取るが、メソッド内に使わない」を表すためです。
```suggestion
          .catch(function (_error) {
```